cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,44 @@
 #!/bin/sh
```

Comment:
So I don't think we want to run POSIX shell. If we have `bash`, let's use it.

Reply:
Done. CP/Paste error.